homeassistant/components/media_player/services.yaml

```diff
@@ -242,6 +242,30 @@ sonos_set_option:
       description: Enable Speech Enhancement mode
       example: 'true'
 
+channels_seek_forward:
```

Comment:
These should be indented under the platform name, ie `channels`.

Reply:
Could you give some more info? I thought I matched the format of the other ones here, like the Sonos services.

Reply:
Sorry, I'm incorrect. This changed after the lazy loading of service descriptions was implemented, I think. So, should be good like you've done it.